lib/jekyll/commands/post.rb

```diff
@@ -53,6 +53,11 @@ def file_name
         def _date_stamp
           @params.date.strftime '%Y-%m-%d'
         end
+
+        def content
```

Comment:
Subclasses should accept the same number of arguments as the superclass so this should be `def content(front_matter = {})`.

```ruby
def content(front_matter = {})
  front_matter.merge!({"date" => @params.data.strftime("%Y-%m-%d %H:%M %z")})
  super
end
```

That should work.

Reply:
@xianny can you update this please :)?